src/transactions/feed/JumpstartTransactionDetailsScreen.tsx

```diff
@@ -0,0 +1,323 @@
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useSelector } from 'react-redux'
+import walletJumpstart from 'src/abis/IWalletJumpstart'
+import BackButton from 'src/components/BackButton'
+import BottomSheet, { BottomSheetRefType } from 'src/components/BottomSheet'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import DataFieldWithCopy from 'src/components/DataFieldWithCopy'
+import LineItemRow from 'src/components/LineItemRow'
+import TokenDisplay from 'src/components/TokenDisplay'
+import TokenIcon, { IconSize } from 'src/components/TokenIcon'
+import CustomHeader from 'src/components/header/CustomHeader'
+import Checkmark from 'src/icons/Checkmark'
+import Logo from 'src/icons/Logo'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { TAG } from 'src/send/saga'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import { useTokenInfo } from 'src/tokens/hooks'
+import TransactionDetails from 'src/transactions/feed/TransactionDetails'
+import NetworkFeeRowItem from 'src/transactions/feed/detailContent/NetworkFeeRowItem'
+import { NetworkId, TokenTransactionTypeV2 } from 'src/transactions/types'
+import Logger from 'src/utils/Logger'
+import { publicClient } from 'src/viem'
+import { SerializableTransactionRequest } from 'src/viem/preparedTransactionSerialization'
+import EstimatedNetworkFee from 'src/walletConnect/screens/EstimatedNetworkFee'
+import { networkIdToNetwork } from 'src/web3/networkConfig'
+import { walletAddressSelector } from 'src/web3/selectors'
+import { Address, Hash, encodeFunctionData, parseEventLogs } from 'viem'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.JumpstartTransactionDetailsScreen>
+
+async function getClaimDataAndStatus(
+  jumpstartContractAddress: Address,
+  networkId: NetworkId,
+  transactionHash: Hash
+) {
+  const viemClient = publicClient[networkIdToNetwork[networkId]]
+
+  const transactionReceipt = await viemClient.getTransactionReceipt({
+    hash: transactionHash,
+  })
+
+  const parsedLogs = parseEventLogs({
+    abi: walletJumpstart.abi,
+    eventName: ['ERC20Deposited'],
+    logs: transactionReceipt.logs,
+  })
+
+  if (parsedLogs.length != 1) {
+    throw new Error('Unexpected number of matching logs')
+  }
+
+  const { beneficiary, index } = parsedLogs[0].args
+
+  Logger.debug('Decoded event', { beneficiary, index })
+
+  const erc20Claim = await viemClient.readContract({
+    address: jumpstartContractAddress as Address,
+    abi: walletJumpstart.abi,
+    functionName: 'erc20Claims',
+    args: [beneficiary, index],
+  })
+
+  const claimed = erc20Claim[3]
+
+  Logger.debug(`Reward deposited in ${transactionHash} ${claimed ? 'was' : 'was NOT'} claimed`)
```

Comment:
nit: to make the logs easier to read, can we add a `TAG` in all the log statements?